LoopBackTests/server/index.js

```diff
@@ -68,7 +80,21 @@ Widget.destroyAll(function () {
   Widget.create({
     name: 'Bar',
     bars: 1,
-    date: '2000-01-02T03:04:05.006Z'
+    data: {
+      data1: 1,
+      data2: 2
+    },
+    array: [
+      'one',
+      'two',
+      'three'
+    ],
+    date: '1970-01-01T00:00:00.000Z',
+    buffer: [12, 34, 56],
```

Comment:
Oh, so it looks like you did actually add a test where the `buffer` value is created/parsed by the server. This code is testing how LoopBack properties of type `Buffer` are handled. My comment above is referring to the way how strong-remoting handles method arguments of type `Buffer`.